core/arch/aarchxx/mangle.c

```diff
@@ -88,9 +90,33 @@ mangle_arch_init(void)
 #ifdef AARCH64
     /* Check address of "lock" is unaligned. See comment in icache_op_struct_t. */
     ASSERT(!ALIGNED(&icache_op_struct.lock, 16));
+#    ifdef DR_HOST_NOT_TARGET
```

Comment:
nit: The #ifdef should be aligned at the beginning of the line. See line 88.

Reply:
This is actually the correct style: we indent nested ifdefs in this manner (# still on 1st column).  This is enforced by clang-format so any deviation here would fail the tests (unless inside one of the places where we disable clang-format but this is not one of them).

Reply:
Yes, it's enforced by clang-format.